src/opentelemetry.ml

```diff
@@ -203,20 +203,19 @@ end = struct
   let of_hex s = of_bytes (Util_.bytes_of_hex s)
 end
 
+module Conventions = struct
```

Comment:
should this have a docstring? to indicate it's default values and parsing of env vars